### DISCUSSION TITLE
AWS Firecracker

### DISCUSSION THREAD

**Discussion by jwinarske** [OE4T/tegra-demo-distro#106] [Show and tell]
I have a layer started for AWS firecracker related recipes.
Builds for Jetson Xavier

https://github.com/jwinarske/meta-microvm

**Comment by ichergui** [comment]
Hey @jwinarske 

Thanks for sharing your repo.
Did you test these recipe with all jetson or only `Xavier` ?

**Reply by jwinarske** [reply]
I will run tests on Nano and Xavier in the next few days.  I'm being asked to submit PR to meta-aws, so it should get more visibility and testing soon.

**Reply by ichergui** [reply]
Hey @jwinarske 
Have you any updates about this topic ?

**Reply by jwinarske** [reply]
I have it running on Xavier and Nano.  Need to generate a PR when I have some free time.  AWS is attempting to validate it using QEMU prior to merge.  I'm not sure QEMU emulate HVA.  So that's the current hold-up.
